CONTRIBUTING.md

```diff
@@ -75,3 +75,28 @@ npm run install:packages --force-local
 
 ## The package publish, deprecation, and alpha publication process
 <!-- TODO: Add this, also consider where to mention npm linking, either here or in the packages README -->
+### Alpha Components
+
+#### Description
+
+The purpose of an Alpha Component is to allow an engineer to work on a component, merge it into Psammead and publish it to npm before it is ready for use in production.
+
+#### Use Cases
+
+The accepted uses of an Alpha Component are:
+
+- When engineering do not have all of the acceptance or accessibility criteria yet but want to make a start on building the component (with the agreement they may need to change it based on the finalised criteria).
+- When a component needs to be merged into Psammead and published, then pulled into Simorgh in order to UX review or a11y test it properly.
+- This means that an Alpha component does not need a UX review or a11y swarm when it is first merged into Psammead, only when it needs to come out of Alpha.
+
+#### Workflow
+
+- A new component is created in Psammead, the version is set to `0.1.0-alpha.1`, and "tag": "alpha" is added to the publish config in the package.json, e.g [alpha tag example](https://github.com/bbc/psammead/commit/64d7aa18a6b6d0861c9fdcd1b88047e634376bb7#diff-19d4d1f1939749aaffae0b8080cc09f4R37-R38)
```

Comment:
When a new component is created should the version be `0.1.0-alpha.1`? Because for now we have been assigning version `1.0.0-alpha.0`. Also maybe we should add a point of making an already existing component into an alpha version (e.g. like it was done with story promos https://github.com/bbc/psammead/blob/latest/packages/components/psammead-story-promo/CHANGELOG.md#psammead-story-promo-changelog).

Reply:
Yep we're going with 0.1.0-alpha.1 for new components, 0.1 indicates it is pre-release and I couldn't find anyone else using alpha.0 for some reason!

I will figure out what the workflow should be for existing components

Reply:
I'm still working out the best workflow for existing components, so that will go in a new PR.